.drone.star

```diff
@@ -100,19 +100,20 @@ def apiTests(ctx):
           'TEST_OCIS':'true',
           'OCIS_REVA_DATA_ROOT': '/srv/app/tmp/reva/',
           'SKELETON_DIR': '/srv/app/tmp/testing/data/apiSkeleton'
-         },
-         'commands': [
-           'git clone -b master --depth=1 https://github.com/owncloud/testing.git /srv/app/tmp/testing',
-           'git clone -b master --depth=1 https://github.com/owncloud/core.git /srv/app/testrunner',
-           'cd /srv/app/testrunner',
-           'make test-acceptance-api'
-          ],
-          'volumes': [
-            {
-              'name': 'gopath',
-              'path': '/srv/app',
-            },
-          ]
+        },
+        'commands': [
+          'git clone -b master --depth=1 https://github.com/owncloud/testing.git /srv/app/tmp/testing',
+          'git clone -b %s --single-branch --no-tags https://github.com/owncloud/core.git /srv/app/testrunner' % (coreBranch),
+          'cd /srv/app/testrunner',
+          'git checkout %s' % (coreCommit),
```

Comment:
can we make it to ignore the commit hash when it is not given, for eg when you want to run just on the latest commit of some branch.

Reply:
You can specify the branch name again, and it will `git checkout master`

Is that good enough?

Reply:
or we can just add something like
```
if (coreCommit == None):
    coreCommit = coreBranch
```
it would look it better IMHO

Reply:
I will fix it properly - it is easy!